backend/db/ddl-12-cset_members_items_plus.jinja.sql

```diff
@@ -1,11 +1,9 @@
 -- Table view: cset_members_items_plus ---------------------------------------------------------------------------------
 CREATE OR REPLACE VIEW {{schema}}cset_members_items_plus{{optional_suffix}} AS (
 SELECT csmi.*
-        , c.vocabulary_id
```

Comment:
## Bugfix: `cset_members_items_plus`
Duplicate columns: `vocabulary_id` and `standard_concept` existed in both tables being joined, so I needed to fix.

## `cset_members_items_plus` missing from DB?
@Sigfried I also noticed that `cset_members_items_plus` was missing from the DB. This shouldn't happen as a result of my refresh. It doesn't delete a table/view until the new one is created. Did you remove it (intentionally)? Do we still need it?